### DISCUSSION TITLE
[@types/chrome] Incorrect return type for chrome.contextMenus.create

### DISCUSSION THREAD

**Discussion by chan-dev** [DefinitelyTyped/DefinitelyTyped#57687] [Issues with a @types package]
Looking at the documentation, it is expected that the method `chrome.contextMenus.create` is supposed to return a number or string, but in the declaration file it returns void.

https://developer.chrome.com/docs/extensions/reference/contextMenus/#method-create

[a comment by an automated account is not shown]

**Comment by nicolas377** [comment]
Thanks for pointing this out! I'll open a PR for this.

**Reply by nicolas377** [reply]
#57688 is open with the fix to this.
